src/runtime/modules/spreadProperties.js

```diff
@@ -0,0 +1,57 @@
+// Copyright 2016 Traceur Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const {
+  defineProperty,
+  getOwnPropertyNames,
+  getOwnPropertySymbols,
+  propertyIsEnumerable,
+} = Object;
+
+function createDataProperty(o, p, v) {
+  defineProperty(o, p, {
+    configurable: true,
+    enumerable: true,
+    value: v,
+    writable: true,
+  });
+}
+
+function copyDataProperties(target, source) {
+  if (source == null) {  // needs to be ==
+    return;
+  }
+  // toObject is called by getOwnProperty*
```

Comment:
Maybe this comment is related to the conversion operation, but the two are a bit cryptic this way.